.mozconfig

```diff
@@ -48,6 +48,8 @@ ac_add_options --disable-tests
 ac_add_options --disable-verify-mar
 
 ac_add_options --enable-application=browser
+ac_add_options --enable-release
+ac_add_options --enable-rust-simd # on x86 requires SSE2
```

Comment:
`-C target-cpu=native` maybe broken for Rust >= 1.24, see [bug 1433747](https://bugzilla.mozilla.org/show_bug.cgi?id=1433747). A corner case unlikely to affect downstream.